esports_data_from_rimble/main.go

```diff
@@ -119,23 +119,16 @@ func matchWinnerFromRimble(inputPtr uint64, secretPtr uint64) uint64 {
 type TeamKillDiff struct {
 	MatchID  string
 	Date     string
-	MapName  string
 	Team1    string
 	Team2    string
 	KillDiff int
 }
 
-func TeamKillDifferenceOnMap(
+func TeamKillDifference(
```

Comment:
The one change I made to the example is that the "team kill diff" is now calculated for all games in a match as opposed to all games on a particular map. I chose to do this to limit the number of env vars the user has to setup. They now have to pass in api key, match id, and date. While they could also pass in a map name I figured this still version of kill diff still gets the point across and doesn't require another env variable